### ISSUE TITLE
Add Beetle Saturn to RALibretro

### ISSUE THREAD

**Issue by magusLOGS** [RetroAchievements/RALibretro#114]
Tested the Core a while back in RALibRetro and it worked, it just needs some BIOS files.

Core:
- mednafen_saturn_libretro.dll

Bios:
- mpr-17933.bin
- saturn_bios.bin
- sega_100.bin
- sega_100a.bin
- sega_101.bin
- sega1003.bin

**Comment by televandalist** [comment]
Commenting here to get it back on the RAWeb channel because I accidentally deleted the post. 

Also, Saturn emulation is so much more pleasant than it was ten, even five years ago.

**Comment by Jamiras** [comment]
I had issues when trying to load a cue file with attached mp3s. RALibRetro would crash with the following message in the log:
```
[ERROR] Unsupported track format: MP3
```
After removing the mp3 tracks from my cue, I was able to successfully load the game. I experienced similar behavior with three other games.

It looks like Saturn discs use a boot method very similar to the Sega CD - there's a 256 byte header in sector 0, then arbitrary code to load the primary executable. As such, the hashing method will most likely be just the header.
